db/base/templatetags/tags.py

```diff
@@ -17,6 +17,6 @@ def frq(value):
         to_format = float(value)
     except (TypeError, ValueError):
         return ''
-    formatted = format(float(to_format) / 1000000, '.3f')
+    formatted = format(float(to_format) / 1000000, '.4f')
```

Comment:
There needs to be a way to truncate leading zeros here. 4 floating points looks bad if we have trailing zeros.